invenio_rdm_records/ext.py

```diff
@@ -212,6 +216,14 @@ def init_resource(self, app):
             config=IIIFResourceConfig,
         )
 
+    def init_registries(self, app):
+        """Initialize registries."""
+        self.custom_fields_registry = CustomFieldsRegistry()
+        for field_key, field_type in app.config.get(
```

Comment:
see above, although we are consistently:
- creating registry
- registering content 

which is what we do both for services and indexers